package.json

```diff
@@ -22,16 +22,34 @@
     "lib",
     "protos"
   ],
-  "main": "lib/v1/index.js",
+  "main": "lib/v2/index.js",
   "bin": {
     "firebase-functions": "./lib/bin/firebase-functions.js"
   },
-  "types": "lib/v1/index.d.ts",
+  "types": "lib/v2/index.d.ts",
   "exports": {
-    ".": "./lib/v1/index.js",
+    ".": "./lib/v2/index.js",
     "./logger/compat": "./lib/logger/compat.js",
     "./logger": "./lib/logger/index.js",
     "./params": "./lib/params/index.js",
+    "./core": "./lib/v2/core.js",
```

Comment:
Wow! I can’t believe we missed these! Can we collocate v1 vs v2?

Reply:
can you clarify what you mean?